src/Controller/UserController.php

```diff
@@ -62,6 +63,7 @@ public function changePassword(
         #[CurrentUser] User $user,
         Request $request,
         EntityManagerInterface $entityManager,
+        #[Autowire('@security.logout_url_generator')]
```

Comment:
Do we really need this? We're using the default logout URL generator service, right?

Reply:
The question is whether Symfony makes it auto-wirable

Reply:
Thanks for checking this 🙏